ungleich_page/models.py

```diff
@@ -124,6 +124,27 @@ class UngleichHeader(CMSPlugin):
     carousel_data_interval = models.IntegerField(default=5000)
 
 
+class UngleichHeaderWithBackgroundImageSliderItem(CMSPlugin):
+    background_image = FilerImageField(
+        null=True, blank=True,
+        related_name="ungleich_header_slider_item_image",
+        on_delete=models.SET_NULL
+    )
+    description = HTMLField(
```

Comment:
I'm not very sure of it, but will it be better to split the description field into separate title, description and link fields? Because the included html will have to be same for all the items.

Reply:
Not me either. Upto @verysanghee to decide what would be the best.
I think an HTMLField is more generic and gives us the freedom of choosing the items that we want.

Reply:
Yes, i agree that it should be easier as of now with HTMLField. But I think if we use elements with `class` attribute in HTMLField it'll mix the data and presentation logic. For any style changes for such a `class` on any HTML element saved in the db, we'll have to find and check/fix each entry.

So, I think ideally we will have to design the html/css structure of a content inside any such HTMLField so that only plain semantic elements (`h2`,`p`,`<strong>`,`<em>`,etc) could provide the style control. If you agree, I can start working on such modifications in a separate ticket.

Reply:
Sure, if you think it would be easier that way, please go ahead.